src/views/order_view.js

```diff
@@ -0,0 +1,73 @@
+import Backbone from 'backbone';
+import Order from '../models/order';
+import Quote from '../models/quote';
+
+const OrderView = Backbone.View.extend({
+  initialize(params) {
+    this.template = params.template;
+    this.currentQuote = params.currentQuote;
+    this.buy = params.buy;
+
+    this.listenTo(this.currentQuote, 'change', this.checkPrice);
```

Comment:
I think this is what's causing the repeat order execution that I mentioned in my feedback notes.

Because the `OrderView` itself is listening to the `Quote` model, we end up getting multiple callbacks setup for the `change` event. The reason for that is actually in the `render` function in `OrderListView`. Each time that view gets re-rendered (for example, because a new order is created)a  completely new set of`OrderView` instances are generated -- but the old instances stick around (invisible because they've been cleared from the page).

For example, if you create a buy order for the HELLO stock at $100 and then another buy order for CLOTH at $95, you will end up with two `OrderView` instances listening to the HELLO quote.

This can be resolved by having `OrderListView` keep an array of all `OrderView` instances it creates, and call `remove()` on them at the beginning of its `render()` function. Another option would be to have the `Order` model itself listen to the `Quote` model's `change` event -- this is my preferred approach.